assets/translations/zh-hans.yaml

```diff
@@ -4916,170 +5976,295 @@ unit:
+    'ui-banners': 此设置使用包含 3 个根参数的 JSON 对象，分别是<code>banner</code>、<code>showHeader</code>和<code>showFooter</code>。<code>banner</code> 是包含 <code>textColor</code>，<code>background</code>和<code>text</code>的对象，其中<code>textColor</code> 和 <code>background</code>的值可以是任何有效的 CSS 颜色值。
   enum:
-    'har-ui-source':
-      auto: 'Auto'
-      bundled: 'Bundled'
-      external: 'External'
-    'har-log-level':
-      info: Info
-      debug: Debug
-      trace: Trace
-    #'ui-default-landing':
-     # ember: Cluster Manager
-      #vue: Cluster Explorer
-    #'telemetry-opt':
-      #prompt: Prompt
-      #in: Opt-in to Telemetry
-      #out: Opt-out of Telemetry
+    'ui-default-landing':
+      ember: Cluster Manager
+      vue: 集群浏览器
+    'telemetry-opt':
+      prompt: 提示
+      in: 加入遥测
+      out: 退出遥测
+    'ui-offline-preferred':
+      dynamic: 动态
+      true: 本地
+      false: 远端
+    'harv-ui-source':
+      auto: '自动'
+      bundled: '包'
+      external: '外部'
+    'harv-log-level':
+      info: 信息
+      debug: 调试
+      trace: 跟踪
 
 featureFlags:
-  label: 功能标志
-  warning: 功能标志允许Rancher将某些功能关在标志后面。你应该谨慎地启用这些功能，它们应该被视为测试版功能，可能会给你的系统带来问题。此外，有些功能（非动态）需要重新启动Rancher服务器才能启用。改变非动态功能将重新启动Rancher pods，这将导致短暂的停电。
-  promptActivate: 请确认您要激活功能标志"{flag}"。
-  promptDeactivate: 请确认你想停用功能标志"{flag}"。
-  restartRequired: "注意：更新该功能标志需要重新启动"
+  label: 功能开关
+  warning: |-
+    功能开关允许 {vendor} 将某些功能隐藏在开关后面。
+    默认关闭的功能是实验功能。
+    某些功能需要重启 {vendor} server 才能更改。
+    这将导致 API 和 UI 短暂中断，但不会影响正在运行的集群或工作负载。
+  promptActivate: 确认是否要激活功能开关“{flag}”
+  promptDeactivate: 确认是否要停用功能开关“{flag}”
+  restartRequired: "注意：你需要重启以更新此功能开关。"
+  restart:
+    title: 等待重启
+    wait: 重启需要数分钟。
+
+branding:
+  label: 公司品牌
+  directoryName: 品牌资产目录名称
+  logos:
+    label: Logo
+    tip: '上传 Logo 以替换顶层导航标题中的 Rancher logo。图片的高度应为 21 像素，最大宽度为 200 像素。最大文件大小为 20KB'
+    lightPreview: 浅色主题预览
+    darkPreview: 深色主题预览
+    uploadLight: 上传浅色 Logo
+    uploadDark: 上传深色 Logo
+    useCustom: 使用自定义 Logo
+  options:
+    default: 默认 Rancher 主题
+    suse: SUSE 主题
+    custom: 定义自定义主题
+  uiPL:
+    label: 私有标签公司名称
+  uiIssues:
+    label: 支持链接
+    issuesUrl: 问题报告 URL
+    invalidUrl: 必须是包括协议头的完整 URL。
+    communityLinks: 显示 Rancher 社区支持链接
+  uiBanner:
+    label: 固定横幅
+    text: 文本
+    textColor: 文本颜色
+    background: 背景颜色
+    showHeader: 在页眉中显示横幅
+    showFooter: 在页脚中显示横幅
+    showConsent: 在登录页面显示同意横幅
+    bannerAlignment:
+      label: 文字对齐
+      leftOption: 左对齐
+      centerOption: 居中
+      rightOption: 右对齐
+    bannerDecoration:
+      label: 文字装饰
+      bannerBold: 加粗
+      bannerItalic: 斜体
+      bannerUnderline: 下划线
+    bannerFontSize:
+      label: '字体大小'
+    consent: 同意横幅
+  color:
+    label: 主颜色
+    tip: 使用自定义颜色替换整个 UI 中使用的主颜色。
+    useCustom: 使用自定义颜色
+  linkColor:
+    label: 链接颜色
+    tip: 使用自定义链接颜色替换整个 UI 中使用的链接颜色。
+    useCustom: 使用自定义链接颜色
+    example: 链接示例
 
```

Comment:
seems like this empty line can be removed

Reply:
The English version also has this line~

Reply:
Sounds good. Keep it then. Other LGTM :)